cachito/web/static/api_v1.yaml

```diff
@@ -422,6 +422,46 @@ paths:
                   error:
                     type: string
                     example: "The request must be in the \"complete\" state before downloading the archive"
+  "/requests/{id}/logs":
+    get:
+      description: Return the logs for a specific Cachito request
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: The ID of the Cachito request to retrieve the logs for
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: The logs for the Cachito request
+          content:
+            text/plain:
+              schema:
+                type: string
+              example: |-
+                Processing Cachito request 1...
+                Processing request...
+                Done processing Cachito request 1
+        '404':
+          description: Logs for Cachito requests is not enabled in Cachito
```

Comment:
This is also returned if Cachito can't find the request or the log file is missing.